src/admin/site_environments.rst

```diff
@@ -0,0 +1,146 @@
+Site Environments
+=================
+
+The site environments required for running PanDA jobs.
+
+Export LSST_LOCAL_PROLOG
+------------------------
+
+For running PanDA jobs, the site only needs to export LSST_LOCAL_PROLOG.
+When LSST_LOCAL_PROLOG is defined, lsst pilot wrapper will source
+LSST_LOCAL_PROLOG during running. In this way, it will not affect other experiments.
+LSST_LOCAL_PROLOG can be defined in .bashrc. for example::
+
+  export LSST_LOCAL_PROLOG=/sdf/home/l/lsstsvc1/.lsst/local_prolog.sh
+
+The content of LSST_LOCAL_PROLOG
+--------------------------------
+
+Here are the content of LSST_LOCAL_PROLOG and corresponding descriptions.
+
+- Timezone:
+
+  To set the timezone, it's good for logging format for jobs from different sites::
+
+    export TZ=UTC
+
+- Realtime logging:
+
+  Environments required for realtime logging::
+
+    # set USE_REALTIME_LOGGING to no if no realtime logging.
+    export REALTIME_LOGGING_SERVER=google-cloud-logging
+    export REALTIME_LOGNAME=Panda-RubinLog
+    export REALTIME_LOGFILES=payload-log.json
+    export USE_REALTIME_LOGGING=yes
+
+- voms and x509 certificates:
+
+  Setup voms-proxy-info and x509 CA certificates (/etc/grid-security/certificates by default).
```

Comment:
What are these usually used for? job submission?

Reply:
it's used for pilot authentication.